packages/baml/baml_client/type_builder.ts

```
@@ -15,93 +15,92 @@ $ pnpm add @boundaryml/baml
 // @ts-nocheck
 // biome-ignore format: autogenerated code
-import type { FieldType } from '@boundaryml/baml/native'
-import {
-	type ClassBuilder,
-	type ClassViewer,
-	type EnumBuilder,
-	EnumViewer,
-	TypeBuilder as _TypeBuilder,
-} from "@boundaryml/baml/type_builder";
-import { DO_NOT_USE_DIRECTLY_UNLESS_YOU_KNOW_WHAT_YOURE_DOING_RUNTIME } from "./globals";
+import { FieldType } from '@boundaryml/baml/native'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)